src/scss/style.scss

```diff
@@ -0,0 +1,32 @@
+@import '~bootstrap/scss/bootstrap';
+
+.weather-card {
+  h6 {
+    color: $primary;
+  }
+
+  p {
+    span {
+      color: $text-muted;
+      // font-weight: light;
```

Comment:
- [ ] Please remove inline comments. This is not a good practice in writing clean code.